### PR TITLE
[Old] Provide file for Qualified Do/Ado notation; implements `IxDiscard`

### LINKED ISSUE #15: Consider a `bind` export for qualified-do
Adding an extra export of `ibind` as `bind` would allow `ibind` to be used with the qualified-do syntax:

```purs
import Control.Monad.Indexed as Ix

foo = Ix.do
  ...
  ...
```

If this seems fine I'll make a PR

### LINKED ISSUE #18: Implement `IxDiscard`
To mirror `Discard` in Prelude, shouldn't there also be an `IxDiscard` type class? If so, it affects how #17 should be implemented.

### PR DESCRIPTION
**Pull request by JordanMartinez**
Fixes #15 
**Edit:** Fixes #18

Part of me wonders whether this should use a different module name than `Control.Monad.Indexed` because this could also work for `ado` notation.

Perhaps something like `Qualified.Indexed`?

### CODE REVIEW

**Comment by garyb** [comment]
`Control.Monad.Indexed.Syntax` might be another ok name. Hmm 🤔

**Comment by JordanMartinez** [comment]
`Syntax` is a bit overreaching though, isn't it? As a user, wouldn't you think that Syntax would include `:>>=`? Should it?

What about `Qualified.Indexed`? "Qualified" in that this is qualified do/ado and "Indixed" in that it's the indexed version of the monad class hierarchy.

Or `QualifiedNotation.Indexed`

**Comment by garyb** [comment]
Let's just stick with `Control.Monad.Indexed.Qualified`, it makes as much sense as anything else I think.

`IxDiscard` looks good too. Are you happy with your changes in here now?

File: src/Control/Monad/Qualified.purs [hunk 1]
```diff
@@ -0,0 +1,42 @@
+-- | Use this module to rebind `do`/`ado` notation via Qualfieid Do/Ado.
```
Comment (garyb) [inline src/Control/Monad/Qualified.purs]: "Qualfieid" typo :wink:

Comment (JordanMartinez) [reply src/Control/Monad/Qualified.purs]: Whoops! Good catch. 

I've fixed it in this [PR's  branch](https://github.com/JordanMartinez/purescript-indexed-monad/commits/master), but the changes aren't being included in this PR for some reason.

**Comment by JordanMartinez** [comment]
I've added the preifx '[Old]' to this broken PR. Feel free to merge #19 instead. I'm satisfied with these changes.

**Pull request closed without merging**